submission/utils.py

```diff
@@ -0,0 +1,96 @@
+import glob
+import json
+import os
+import shutil
+
+
+def get_new_d3m_path():
+    """
+    Gets the name of the newest version path for d3m/byudml
+    :return:
+    """
+    new_directory = max(glob.glob('submission/primitives/v????.?.?'))
```

Comment:
Something needs to be done about these ?s. It would be nice if the version was single sourced. Maybe check the installed version dynamically.

Reply:
The ?'s are a regex - it finds the biggest value given those numbers.  Do you mean something else about those?

Reply:
Ohm that's great. Thanks! No change needed.